backend/scripts/init_database.py

```
@@ -1,76 +1,121 @@
                 try:
-                    await collection.create_index(
-                        index_config["keys"],
-                        name=index_config["name"],
-                        unique=index_config.get("unique", False),
-                        sparse=index_config.get("sparse", False)
-                    )
-                    print(f"   ✅ Created index: {index_config['name']}")
+                    await collection.create_index(index_spec)
+                    index_name = ", ".join([f"{k}({v})" for k, v in index_spec])
+                    print(f"   ✓ Index created: {index_name}")
                 except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)